src/main/java/io/github/perrymant/moneymaker/Application.java

```diff
@@ -1,9 +1,15 @@
 package io.github.perrymant.moneymaker;
 
+import java.util.ArrayList;
+import java.util.List;
+
 class Application {
     static final String ERROR_MESSAGE = "Error: Invalid argument: moneymaker help for more info.";
     private final Budget budget;
     private final FileReader reader = new FileReader();
+    private final List<Transaction> reportJSON = new ArrayList<>();
```

Comment:
This, and the imports above do nothing. Let's delete it.

Reply:
Yep - sorry, this was me starting on the next sprint task... will remove.